amgut/lib/data_access/ag_data_access.py

```diff
@@ -412,6 +412,11 @@ def deleteAGParticipant(self, ag_login_id, participant_name):
                    "survey_id = %s")
             curr.execute(sql, [survey_id])
 
+            # Reset survey attached to barcode(s)
+            sql = ("UPDATE ag_kit_barcodes SET survey_id = NULL WHERE "
```

Comment:
`survey_id` is nullable in `ag_kit_barcodes` as unassigned barcodes are not associated with a valid survey